tests/components/ecovacs/test_select.py

```diff
@@ -47,6 +49,12 @@ async def notify_events(hass: HomeAssistant, event_bus: EventBus):
                 "select.ozmo_950_water_flow_level",
             ],
         ),
+        (
+            "n0vyif",
```

Comment:
Test was added to ensure that the integrations is also working after the breaking change